### PR TITLE
Ex. 5 SQS

### PR DESCRIPTION
**Pull request by kelseykodes**
attempt 1

### CODE REVIEW

File: Assignments/stu7/assignments.py [hunk 1]
```diff
@@ -128,3 +129,66 @@ def get_people(people_list):
...
 
 
+# Ex5:
+def send_message_to_sqs(cat, QUEUE_URL):
+
+    response = sqs.send_message(
+        QueueUrl=QUEUE_URL,
+        DelaySeconds=10,
+        MessageAttributes={
+            'UserName': {
```
Comment (mburolla) [inline Assignments/stu7/assignments.py]: We should change UserName to be CatName.

File: Assignments/stu7/assignments.py [hunk 2]
```diff
@@ -128,3 +129,66 @@ def get_people(people_list):
...
+                'DataType': 'String',
+                'StringValue': cat['cat_name']
+            },
+            'UserId': {
```
Comment (mburolla) [inline Assignments/stu7/assignments.py]: Also change UserId to be CatId.

File: Assignments/stu7/assignments.py [hunk 3]
```diff
@@ -128,3 +129,66 @@ def get_people(people_list):
...
+    message = None
+    try:
+        message = response['Messages'][0]  # Only read one.
+    except KeyError as ke:
+        logging.info("SQS queue is empty.")
+
+    if message:
+        retval = {
+            "status": message["Body"],
+            "cat_id": message["MessageAttributes"]["UserId"]["StringValue"],
```
Comment (mburolla) [inline Assignments/stu7/assignments.py]: And change UserId and UserName here...

Comment (kelseykodes) [reply Assignments/stu7/assignments.py]: i believe it's been corrected now

**Pull request merged**